d2l-button-icon.html

```diff
@@ -95,14 +104,47 @@
 				text: {
 					type: String,
 					reflectToAttribute: true
+				},
+
+				/**
+				 * Horizontal alignment of button. Options:
+				 * 	"text" -  The button icon will left align with the page content
+				 *	default - The button's edge (including padding) will left align with the page content
+				*/
+				hAlign: {
+					type: String,
+					reflectToAttribute: true
+				},
+
+				isRtl: {
+					type: Boolean,
+					value: false,
+					computed: '_calculateRTL(isAttached)',
+					reflectToAttribute: true
 				}
 
 			},
 
 			behaviors: [
 				D2L.PolymerBehaviors.Button.Behavior,
 				D2L.PolymerBehaviors.FocusableBehavior
-			]
+			],
+
+			_calculateRTL: function(isAttached) {
```

Comment:
If it turns out that we do need to calculate direction, maybe we can just get away [something like this](https://github.com/BrightspaceUI/dropdown/blob/master/d2l-dropdown-content-behavior.html#L516)?